### PR TITLE
feat(presets): Group for Spock Framework packages

### PR DESCRIPTION
**Pull request by szpak**
## Changes

[Several](https://search.maven.org/search?q=g:org.spockframework) Spock Framework packages should be updates at the same time.

## Context

When a dedicated prperty is not used, multiple modules of Spock used in the same projects could be updated in separate MRs, which doesn't make sense.

## Documentation (please check one with an [x])

- [ ] I have updated the documentation, or
- [x] No documentation update is required

## How I've tested my work (please select one)

I have verified these changes via:

- [x] Code inspection only, or
- [ ] Newly added/modified unit tests, or
- [x] No unit tests but ran on a real repository, or
- [ ] Both unit tests + ran on a real repository

### CODE REVIEW

**Comment by szpak** [comment]
(force push to fix typo in the commit message...)

**Review by viceice — CHANGES_REQUESTED** [review]
please checkout monorepo.ts for better grouping.

**Comment by szpak** [comment]
@viceice I made the changes. Is it something you expected?

Unfortunately, I don't know how to test it (the regex) in practice (without rebuilding Renovate locally)? Should I write a test case in `monorepo.spec.ts` or "it should be already covered"?

**Comment by szpak** [comment]
Btw, if the `monorepo.ts` is a recommended place to handle those cases, maybe it would be good to put a comment in the `group.ts` to redirect people modifying that file to the "new" one?

File: lib/config/presets/internal/monorepo.ts [hunk 1]
```diff
@@ -271,6 +271,7 @@ const patternGroups = {
   embroider: '^@embroider/',
   fullcalendar: '^@fullcalendar/',
   spfx: ['^@microsoft\\/sp-', '^@microsoft\\/eslint-.+-spfx$'],
+  spock: '^org.spockframework:spock-',
```
Comment (viceice) [inline lib/config/presets/internal/monorepo.ts:274]:
```suggestion
  spock: '^org\\.spockframework:spock-',
```
escape the dot

**Review by viceice — APPROVED** [review]

**Pull request merged**